whipper/command/mblookup.py

```diff
@@ -1,46 +1,57 @@
 from whipper.command.basecommand import BaseCommand
-from whipper.common.mbngs import musicbrainz
+from whipper.common.mbngs import musicbrainz, getReleaseMetadata
 
 
 class MBLookup(BaseCommand):
     summary = "lookup MusicBrainz entry"
-    description = """Look up a MusicBrainz disc id and output information.
+    description = """
+Look up a MusicBrainz disc id or release id and output information.
 
 You can get the MusicBrainz disc id with whipper cd info.
 
-Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-"""
+Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-
+"""
 
     def add_arguments(self):
-        self.parser.add_argument(
-            'mbdiscid', action='store', help="MB disc id to look up"
-        )
+        self.parser.add_argument('-d', '--disc-id',
+                                 action='store', dest='disc_id',
+                                 help="MB disc id to look up", default=None)
+        self.parser.add_argument('-R', '--release-id',
+                                 action='store', dest='release_id',
+                                 help="MB release id to look up", default=None)
+
+    def _printMetadata(self, md):
+        print('    Artist: %s' % md.artist.encode('utf-8'))
+        print('    Title:  %s' % md.title.encode('utf-8'))
+        print('    Type:   %s' % str(md.releaseType).encode('utf-8'))  # noqa: E501
```

Comment:
Is the `noqa` comment still needed now that it’s been unindented a bit?

Reply:
I have removed `noqa` in a new commit. Thanks !